tests/test_pipelines_summarization.py

```diff
@@ -46,6 +46,7 @@ def test_input_too_long(self):
             decoder_attention_heads=1,
             max_length=4,
             min_length=1,
+            forced_eos_token_id=None,
```

Comment:
The test uses `greedy_search` and eos was not forced in `greedy_search` before this PR